docs/getting-started/asgi.md

```diff
@@ -0,0 +1,62 @@
+# ASGI
+
+Wrap your ASGI application inside `django_user_trace.asgi.UserTraceMiddleware`
+and [`channels.auth.AuthMiddlewareStack`][channels:authentication].
+
+```py
+import os
+
+from asgi_correlation_id.middleware import CorrelationIdMiddleware
+from channels.auth import AuthMiddlewareStack
+from channels.routing import ProtocolTypeRouter, URLRouter
+from django.core.asgi import get_asgi_application
+from django.urls import re_path
+from django_user_trace.asgi import UserTraceMiddleware
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "example.settings")
+
+django_asgi_application = get_asgi_application()
+
+application = ProtocolTypeRouter(
+    {
+        "http": CorrelationIdMiddleware(#(2)!
+            AuthMiddlewareStack(
```

Comment:
We probably shouldn't apply the auth middleware stack to the Django ASGI application, as it will do its own middleware/authentication.